tests/ui/conftest.py

```diff
@@ -47,61 +60,38 @@ def selenium(selenium, request):
     Mobile size: 738x414 (iPhone 7+)
 
     """
+    # Skip mobile test with marker 'desktoponly'
+    if request.node.get_marker('desktoponly') and request.param == MOBILE:
+        pytest.skip('Skipping mobile test')
     selenium.set_window_size(*request.param)
     return selenium
 
 
 @pytest.fixture(scope='session')
-def fxa_account(base_url):
-    """Account used to login to the AMO site."""
-    url = DEV_URL if 'olympia' or 'localhost' in base_url else PROD_URL
-    return FxATestAccount(url)
-
-
-@pytest.fixture(scope='session')
-def jwt_issuer(base_url, variables):
-    """JWT Issuer from variables file or env variable named 'JWT_ISSUER'"""
-    try:
-        hostname = urlparse.urlsplit(base_url).hostname
-        return variables['api'][hostname]['jwt_issuer']
-    except KeyError:
-        return os.getenv('JWT_ISSUER')
-
+def base_url(base_url):
+    return base_url
```

Comment:
Do we actually need this if it's just a dummy like that? :thinking:

Reply:
Nope!

Reply:
Its so lonely 😨